base/src/types.rs

```diff
@@ -351,6 +352,64 @@ pub enum FontScheme {
     None,
 }
 
+// MergedRange type
+// There will be one MergedRange struct maintained for every Merge cells that we load
+// merge_cell_range : Its tuple having [row_start, column_start, row_end, column_end]
+#[derive(Encode, Decode, Debug, PartialEq, Eq, Clone)]
+pub struct MergedRange(i32, i32, i32, i32);
+
+// implementing accessor function
+impl MergedRange {
```

Comment:
I see where you are coming from, and you are not wrong, but IronCalc doesn't do this. We have a solid convention:

`(row_start, column_start, row_end, column_end)`

and we access members directly. What you are doing is great, and has many advantages, but for now I would like to have a more uniform code style.

There are many advantages of what you are doing. For instance all those are valid choices:

```
(row_start, column_start, row_end, column_end)
(row_start, row_end, column_start, column_end)
(row_start, column_start, row_width, column_width)
...
```
and many variations. And we might end up refactoring things in that direction one day